lib/msf/core/exploit/file_dropper.rb

```diff
@@ -4,7 +4,7 @@ module Msf
 module Exploit::FileDropper
 
   def initialize(info = {})
-    super(
+    super(update_info(info,
```

Comment:
Also fixed in https://github.com/rapid7/metasploit-framework/pull/15669/files#r707610383

But will ship independently :+1: